content/docs/2.8/concepts/scaling-deployments.md

```diff
@@ -62,6 +62,7 @@ spec:
           - type: Percent
             value: 100
             periodSeconds: 15
+      name: {name-of-hpa-resource}                   # Optional. Default: keda-hpa-{scaled-object-name}
```

Comment:
I'd move this row to above `behavior` because both are at the same indentation level, and together it's clearer for reading (I had to use a paper as a rect line to check `name` indentations). WDYT?